polyply/src/generate_templates.py

```diff
@@ -90,8 +90,8 @@ def find_interaction_involving(block, current_node, prev_node):
                 elif prev_node in interaction.atoms and inter_type.split("_")[0] == "virtual":
                     return True, interaction, inter_type
     else:
-        msg = ('Cannot build template for residue {}. No constraint, bond, virtual-site'
-               'linking atom {} and atom {}.')
+        msg = '''Cannot build template for residue {}. No constraint, bond, virtual-site
+                 linking atom {} and atom {}.'''
```

Comment:
```suggestion
        msg = 'Cannot build template for residue {}. No constraint, bond, virtual-site '
              'linking atom {} and atom {}.'
```
The downside of the triple quotes is that you also get the newline.